### PR TITLE
Override moment relative timestamp translations for Nepali and Serbian services.

### LINKED ISSUE #2153: Override moment relative timestamp translations - Nepali, Serbian
**Is your feature request related to a problem? Please describe.**
Apply the correct wording for relative timestamp to match what we have currently on PAL.

**Describe the solution you'd like**
Override moment's default relative time in `psammead-locales` to have the correct translations. 
**Services**
- Nepali
- Serbian (cyr)
- Serbian  (lat)

**Example**
```js
moment.updateLocale('yo', {
  relativeTime: {
    past: '%s sẹ́yìn',
    s: 'ìṣẹ́jú aayá die',
    m: 'ìṣẹ́jú kan',
    mm: 'ìṣẹ́jú %d',
    h: 'wákàtí kan',
    hh: 'wákàtí %d',
  },
});
```
More details on the spreadsheet. 

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #2153

**Overall change:** _Override moment relative timestamp translations for Nepali and Serbian services._
**Code changes:**

- _Add relative timestamp overrides._
- _Update Nepali moment month translations._
- _Update tests._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/CHANGELOG.md [hunk 1]
```diff
@@ -3,12 +3,13 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.16.0 | [PR#2210](https://github.com/bbc/psammead/pull/2210) Add moment relative timestamp overrides for `ne`, `sr` and `sr-cyrl` locales |
```
Comment (ghost) [inline packages/utilities/psammead-locales/CHANGELOG.md]:
```suggestion
| 2.16.0 | [PR#2210](https://github.com/bbc/psammead/pull/2210) Add moment relative timestamp overrides for `ne` (Nepali), `sr` and `sr-cyrl` (Serbian) locales |
```

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Review by OlgaLyubin — APPROVED** [review]
LGTM

**Review by ghost — APPROVED** [review]

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**